src/wolf.system/CMakeLists.txt

```diff
@@ -10,51 +9,68 @@ set(LIBRARY_TYPE "SHARED" CACHE STRING "Library type")
 
 if("${TARGET}" STREQUAL "IOS")
     set(IOS TRUE)
-endif()													
+    set(CMAKE_OSX_SYSROOT iphoneos)
+    set(CMAKE_SYSTEM_NAME iOS)
+    set(CMAKE_OSX_ARCHITECTURES ${ARCH})
+    # signing the project
+    #set_xcode_property(${PROJECT_NAME} CODE_SIGN_IDENTITY "Apple Development")  
+    #set_xcode_property(${PROJECT_NAME} DEVELOPMENT_TEAM ${DEVELOPMENT_TEAM_ID})
+endif()
+
 if(UNIX AND NOT APPLE AND NOT IOS)
    set(LINUX TRUE)
 endif()
 
 # set the project name
 if (IOS)
-    project (wolf_system_ios)						 
+    project(wolf_system_ios VERSION 2.1.0)
```

Comment:
Instead of writing the version for each platform separately, please define a variable and set it and then use that variable for all platforms. This would not cause any problem because for every merge we want the release packages for all platforms simultaneously (i.e. there is no case that the version for windows is different from the version of osx and etc.).

For example look at the following codes:

```
set (APPLICATION_VERSION_MAJOR 1)
set (APPLICATION_VERSION_MINOR 0)
set (APPLICATION_VERSION_PATCH 0)
set (APPLICATION_VERSION_STRING "${APPLICATION_VERSION_MAJOR}.${APPLICATION_VERSION_MINOR}.${APPLICATION_VERSION_PATCH}-${APPLICATION_VERSION_TYPE}")

if (IOS)
    project(wolf_system_ios VERSION ${APPLICATION_VERSION_STRING})
elseif (APPLE)
    project(wolf_system_osx VERSION ${APPLICATION_VERSION_STRING})
elseif (WIN32)
    project(wolf_system_win VERSION ${APPLICATION_VERSION_STRING})
elseif (LINUX)
    project(wolf_system_linux VERSION ${APPLICATION_VERSION_STRING})
endif()
```

Reply:
I'll use `WOLF_` instead of `APPLICATION_`.